core/src/main/scala/com/softwaremill/kmq/redelivery/streams/RedeliveryAndCommitMarkerStream.scala

```diff
@@ -0,0 +1,54 @@
+package com.softwaremill.kmq.redelivery.streams
+
+import akka.Done
+import akka.actor.ActorSystem
+import akka.kafka.ConsumerMessage.CommittableMessage
+import akka.kafka.scaladsl.Consumer
+import akka.kafka.scaladsl.Consumer.DrainingControl
+import akka.kafka.{ConsumerSettings, Subscriptions}
+import akka.stream.ClosedShape
+import akka.stream.scaladsl.{Broadcast, Flow, GraphDSL, RunnableGraph, Sink}
+import com.softwaremill.kmq._
+import com.typesafe.scalalogging.StrictLogging
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+ * Combines functionality of [[RedeliveryStream]] and [[CommitMarkerStream]].
+ */
+class RedeliveryAndCommitMarkerStream(markerConsumerSettings: ConsumerSettings[MarkerKey, MarkerValue],
```

Comment:
that's a poor name :) maybe simple `RedeliveryTrackerStream`? It implements the main logic, after all. The name shouldn't enumerate implementation details

Reply:
OK. I'm changing the name to `RedeliveryTrackerStream`.